integration/graphql-types.ts

```diff
@@ -53,8 +63,8 @@ export interface ContainerResolvers {
 }
 
 export interface SubscriptionResolvers {
-  authorSaved: SubscriptionResolver<Subscription, {}, AuthorId>;
-  searchSub: SubscriptionResolver<Subscription, SubscriptionSearchSubArgs, Array<AuthorId | Book>>;
+  authorSaved: Resolver<Subscription, {}, AuthorId>;
```

Comment:
Was this line supposed to change? It looks like `SubscriptionResolver` was special/different before.

Reply:
Whoops! TY - This has been fixed.